cypress/integration/basic-functionalities/local-tests/reporter/1_Create_a_report.spec.js

```diff
@@ -1,105 +1,110 @@
+      cy.get('[data-test-id="button-save"]').click()
+      cy.get('.b-toast-success') // We check if the success toast appears
+      cy.get('[data-test-id="input-name"]').should('have.value', 'Cypress report')
+      cy.get('[data-test-id="input-handle"]').should('have.value', 'cypress_handle')
+      cy.get('[data-test-id="input-description"]').should('have.value', 'This is an automated description.')
+      cy.get('[data-test-id="button-report-builder"]').click()
+      cy.contains('Cypress report')
+      cy.get('[data-test-id="button-back"]').click()
+      cy.get('table > tbody > :first-child()').click() // We click on the created report
+      cy.get('.b-toast-danger').should('not.exist') // We check that an error toast doesn't appear
+      cy.visit(baseURL + '/list') // Visiting main page of Reporter
+    })
+  })
 
-  context("Test for checking if permissions open up when in edit mode", () => {
-    it("should be able to open permissions view", () => {
-      cy.get("table > tbody > :first-child() > :last-child() > a:nth-child(2)").click(); // We click on the Edit button on the created report
-      cy.get('[data-test-id="button-permissions"]').click(); // We click on the Permissions button
-      cy.get(".close").click({ multiple: true }); // Once permissions view open, we close the view
-      cy.get('[data-test-id="button-back"]').click(); // We click on the Back button to go on the main page
-    });
-  });
+  // context('Test for checking if permissions open up when in edit mode', () => {
+  //   it('should be able to open permissions view', () => {
+  //     cy.get('table > tbody > :first-child() > :last-child() > a:nth-child(2)').click() // We click on the Edit button on the created report
+  //     cy.get('[data-test-id="button-permissions"]').click()
+  //     cy.get('.close').click({ multiple: true }) // Once permissions view open, we close the view
+  //     cy.get('[data-test-id="button-back"]').click()
+  //   })
+  // })
 
-  context("Test for checking if new report and delete buttons are present when in edit mode", () => {
-    it("should be able to access new report and delete buttons", () => {
-      cy.get("table > tbody > :first-child() > :last-child() > a:nth-child(2)").click(); // We click on the Edit button on the created report
-      cy.get('[data-test-id="button-create-report"]'); // We check if the New report button is present
-      cy.get('[data-test-id="button-delete"]'); // We check if the Delete button is present
-      cy.get('[data-test-id="button-back"]').click(); // We click on the Back button to go on the main page
-    });
-  });
+  context('Test for checking if new report and delete buttons are present when in edit mode', () => {
+    it('should be able to access new report and delete buttons', () => {
+      cy.get('table > tbody > :first-child() > :last-child() > a:nth-child(2)').click() // We click on the Edit button on the created report
+      cy.get('[data-test-id="button-create-report"]')
```

Comment:
here, where you check if a button is present, is this the correct way to do it? Shouldn't there be a should('have') or something like this?